lib/cadet/assessments/answer_types/mcq_answer.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Cadet.Assessments.AnswerTypes.MCQAnswer do
+  @moduledoc """
+  The Assessments.QuestionTypes.MCQQuestion entity represents an MCQ Answer.
+  It comprises of one of the MCQ choices.
+  """
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Cadet.Assessments.QuestionTypes.MCQChoice
+
+  embedded_schema do
+    field(:choice_id, :integer)  
+  end
+
+  @required_fields ~w(choice_id)a
+
+  def changeset(answer, params \\ %{}) do
+    answer
+    |> cast(params, @required_fields)
+    |> validate_number(:choice_id, greater_than: 0, less_than: 5)
```

Comment:
I don't think we should `validate_number` since it's possible in the future we have like 10 choices in MCQ. Let's not shoot ourselves in the foot by specifying hard limits.